chain/neutrino.go

```diff
@@ -801,9 +801,18 @@ out:
 
 		case err := <-rescanErr:
 			if err != nil {
-				log.Errorf("Neutrino rescan ended with error: %s", err)
+				log.Errorf("Neutrino rescan ended with "+
```

Comment:
Do we know what the error is? What is the root cause?

Reply:
I am trying to obtain the logs from the user, but I think his node is generally working fine, my next step will be to let the user run the patch to see if that fixes the problem. But as Oliver and Laolu pointed out, this code part needs a refactor.